builtin/game/falling.lua

```diff
@@ -42,8 +42,7 @@ core.register_entity(":__builtin:falling_node", {
 
 	on_step = function(self, dtime)
 		 -- Set gravity
-		local acceleration = self.object:getacceleration()
-		if not vector.equals(acceleration, {x = 0, y = -10, z = 0}) then
+		if self.object:getacceleration().y ~= -10 then
```

Comment:
Not identical, checking if the y component is -10 is not the same as checking that the vector is {x = 0, y = -10, z = 0}. Whether this matters i don't know.